### PR TITLE
Dev

### LINKED ISSUE #1: Initialize  the project
# REQUIREMENTS
-Init a new react-native project
-Install ESlint
-Install DeepScan 
-install Code Climate
-Create a firebase database

### LINKED ISSUE #2: SignUp: create  a form
# REQUIREMENTS
-The form must have 5 fields (name, email, pass, terms, suscribe updates)
-Buttom to SignUp
-Button to SignUp with Google
-Option to Log In

# DESIGN
[SignUp design](https://raw.githubusercontent.com/BC-MAY-21-RN/kata-05-reservacion-de-vuelos-k5-f22-t4/master/img/registro.pngl)

### LINKED ISSUE #3: SignUp: Apply Validations form
# REQUIREMENTS
-All fields are full
-The pass must have 8 or more characters with a mix letteres, numbers and symbols.
-The terms must be accepted
-The buttons will be enables when all fields are completes.

### PR DESCRIPTION
**Pull request by Etneciv**
# Description
Init the project in react-native
Depednences added: react-native vector-icons, react-native-bouncy-checkbox, 
Cmponents created: SignUp form, SignUpStyles, colors
Screens created: SignUp screen   

 # Testing
 Help me how can I test or look at the changes
 
  ## Screenshots 
[IMAGE](https://i.imgur.com/mF7hwVX.png)

### CODE REVIEW

File: ReservacionVuelos/.ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.7.4
```
Comment (YilKanda) [inline ReservacionVuelos/.ruby-version:1]: Why is this version of Ruby in your project?

File: ReservacionVuelos/Gemfile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+source 'https://rubygems.org'
+
+# You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
+ruby '2.7.4'
+
+gem 'cocoapods', '~> 1.11', '>= 1.11.2'
```
Comment (YilKanda) [inline ReservacionVuelos/Gemfile]: Why is Ruby's Gemfile in your project?

**Review by YilKanda — COMMENTED** [review]
Remember that each issue is a simple task that needs to be solved, and each Pull Request corresponds to each issue.

I mention it because in this Pull Request you are placing several issues, which is not recommended, such as:

- Init the project in react-native
- Depednences added: react-native vector-icons, react-native-bouncy-checkbox,
- Cmponents created: SignUp form, SignUpStyles, colors
- Screens created: SignUp screen

File: ReservacionVuelos/.ruby-version (hunk 1 above)
Comment (cfcortes) [reply ReservacionVuelos/.ruby-version:1]: @YilKanda @Blueheart90 mentioned that those files came o where generated automatically.. should they be removed?

File: ReservacionVuelos/Gemfile (hunk 2 above)
Comment (Etneciv) [reply ReservacionVuelos/Gemfile:6]: React -native generated the document when we init the project from console

File: ReservacionVuelos/.ruby-version (hunk 1 above)
Comment (jbernalme) [reply ReservacionVuelos/.ruby-version:1]: The Ruby files was removed from project, in this commit d856a3799aab882c4db54c22d5ff1d574b47572e

**Review by CrMa1 — APPROVED** [review]
God Job c:

**Review by Jonathanrrs — APPROVED** [review]
Good job

**Review by YilKanda — APPROVED** [review]

**Pull request merged**